command/operator_usage.go

```diff
@@ -196,8 +196,8 @@ type UsageResponse struct {
 	entityCount   int64
 	// As per 1.9, the tokenCount field will contain the distinct non-entity
 	// token clients instead of each individual token.
-	tokenCount int64
-
+	tokenCount  int64
+	SecretSyncs int64
```

Comment:
Is there a reason this needs to be exported? Not a big deal either way, but the other fields aren't, so I was curious.

Reply:
nope, no reason at all